src/scales/scale.logarithmic.js

```diff
@@ -62,6 +62,11 @@ var defaultConfig = {
 	}
 };
 
+// TODO(v3): change this to isPositive
+function isNonNegative(value) {
+	return typeof value === 'number' && value >= 0;
```

Comment:
Would be more robust using `return helpers.isFinite(value) && value >= 0;`

Reply:
done